VERSION

```diff
@@ -1 +1 @@
-0.4.1
+0.5
```

Comment:
Version bumps are handled outside PRs. (This is also not a valid Factorio mod version number. It would have to be `0.5.0`.)